manim/mobject/mobject.py

```diff
@@ -2039,40 +2040,235 @@ def construct(self):
             self.center()
         return self
 
-    def arrange_in_grid(self, n_rows=None, n_cols=None, **kwargs):
-        """Sorts :class:`~.Mobject` next to each other on screen using a grid.
+    def arrange_in_grid(
+        self,
+        rows: Optional[int] = None,
+        cols: Optional[int] = None,
+        buff: Union[float, Tuple[float, float]] = MED_SMALL_BUFF,
+        cell_alignment: np.ndarray = ORIGIN,
+        row_alignments: Optional[str] = None,  # "ucd"
+        col_alignments: Optional[str] = None,  # "lcr"
+        row_heights: Optional[Iterable[Optional[float]]] = None,
+        col_widths: Optional[Iterable[Optional[float]]] = None,
+        flow_order: str = "rd",
+        **kwargs,
+    ) -> "Mobject":
+        """Arrange submobjects in a grid.
+
+        Parameters
+        ----------
+        rows
+            The number of rows in the grid.
+        cols
+            The number of columns in the grid.
+        buff
+            The gap between grid cells. To specify a different buffer in the horizontal and
+            vertical directions, a tuple of two values can be given - `(row, col)`.
+        cell_alignment
+            The way each submobject is aligned in its grid cell.
+        row_alignments
+            The vertical alignment for each row (top to bottom). Accepts the following characters: `"u"` -
+            up, `"c"` - center, `"d"` - down.
+        col_alignments
+            The horizontal alignment for each column (left to right). Accepts the following characters `"l"` - left,
+             `"c"` - center, `"r"` - right.
```

Comment:
In the rendered page, there is some unfortunate formatting going on with this (I think I might have introduced the error 😅). This should fix it.
```suggestion
        col_alignments
            The horizontal alignment for each column (left to right). Accepts the following characters `"l"` - 
            left, `"c"` - center, `"r"` - right.
```